ui/app/src/views/ViewDashboard.tsx

```diff
@@ -156,9 +156,9 @@ function ViewDashboard() {
                   dashboardProject={data.metadata.project}
                 />
               }
-              emptyDashboard={
-                <EmptyDashboard additionalText="In order to create a new dashboard, you need to add at least one panel!" />
-              }
+              emptyDashboardProps={{
```

Comment:
This works although I slightly prefer being able to control the full component. Maybe, down the road it might make sense to `emptyDashboardComponent` to do that (similar to `dashboardTitleComponent`)